packages/kbn-management/settings/application/hooks/use_settings.ts

```diff
@@ -9,35 +9,25 @@
 import { useState } from 'react';
 import useEffectOnce from 'react-use/lib/useEffectOnce';
 
-import { IUiSettingsClient } from '@kbn/core-ui-settings-browser';
-import { normalizeSettings } from '@kbn/management-settings-utilities';
-import { SettingType, UiSettingMetadata } from '@kbn/management-settings-types';
+import { useServices } from '../services';
 
 /**
  * React hook which retrieves settings from a particular {@link IUiSettingsClient},
  * normalizes them to a predictable format, {@link UiSettingMetadata}, and returns
  * them as an observed collection.
  */
-export const useSettings = (client: IUiSettingsClient) => {
-  const getAllowlistedSettings = (settingsClient: IUiSettingsClient) => {
```

Comment:
This function relies directly on the settings client.  If we move it up to the `Services`, we can abstract it away, (and remove the direct dependency).